modules/cnf-topology-aware-lifecycle-manager-apply-policies.adoc

```diff
@@ -333,7 +333,7 @@ $ oc get clusterversion
 [source,terminal]
```

Comment:
You'll need to add a subs attribute to render attributes.

[source,terminal,subs="attributes+"]

Otherwise, your codeblocks will render as follows:

![Uploading Screenshot from 2024-03-04 15-36-39.png…]()

Reply:
If you search your preview links you'll see many "{product-version}" entries. These should state branch build until the docs are published.